config/application.rb

```diff
@@ -37,6 +37,8 @@ class Application < Rails::Application
     config.active_record.observers = :order_detail_observer
     #:cacher, :garbage_collector, :forum_observer
 
+    config.active_record.belongs_to_required_by_default = false
```

Comment:
I feel like this belongs in `config/initializers/new_framework_defaults.rb`, but we don't have that file, so I can't say for sure.

https://blog.bigbinary.com/2016/08/18/new-framework-defaults-in-rails-5-to-make-upgrade-easier.html

Reply:
Let's consider this when we go through the `bin/rails app:update` together. The first time I did it I may have missed some things.